Tests/.editorconfig

```diff
@@ -214,6 +214,9 @@ dotnet_diagnostic.MA0096.severity = none
 # A class that implements IComparable<T> or IComparable should override comparison operators. Don't care about this.
 dotnet_diagnostic.MA0097.severity = none
 
+# Do not convert implicitly to DateTimeOffset. Fair point, but we need this in our tests
+dotnet_diagnostic.MA0132.severity = suggestion
```

Comment:
Why do we need the implicitness?
The explicit road worked in my WIP 🙃 
https://github.com/jnyrup/fluentassertions/commit/a2feb8965cec859171f3a815026e02c5c92f6f82

Reply:
I was assuming we were telling people to do just that, but it seems we typically use `WithOffset`. So maybe you're right.